.github/pr-blocking-todos/tests/test_checkfortodos.py

```diff
@@ -0,0 +1,72 @@
+from checkfortodos import checkForTodos, isTodoLine
+
+import unittest
+
+class TestCheckForTodos(unittest.TestCase):
```

Comment:
The unit tests for isTodoLine are very nice.  Should we also have tests for checkForTodos?  I'm inclined to say no, because the main logic is in the function we've already tested.

Reply:
I'd say the code is simple enough and the functionality sufficiently non-critical that we can avoid testing the `checkForTodos` function itself.